### DISCUSSION TITLE
Duda input y output

### DISCUSSION THREAD

**Discussion by ignaciomanzanares** [IIC2133-PUC/2024-1#28] [Tarea 0] [answered]
Nosotros tenemos que crear los archivos input y output ?

**Comment by sachondo7** [comment, marked as answer]
Hola, los tests están subidos en este [link](https://drive.google.com/drive/folders/1SZAYrsW6QzvnxJYK_sqfxS8ZDS6qR7IR) 
Tú puedes crear manualmente el archivo de input y output para ir copiando los tests y luego ir comparando
